packages/next/src/generators/init/utils/createCustomTestConfig.ts

```diff
@@ -0,0 +1,43 @@
+import { joinPathFragments, Tree, updateJson } from '@nrwl/devkit';
```

Comment:
File naming should be kebab-case for consistency.

Reply:
This is my bad. Will update